### ISSUE TITLE
MBAAS Auth user custom role + permission

### ISSUE THREAD

**Issue by fileman** [microsoft/appcenter#477]
**Describe the solution you'd like**
Something similar at asp.net policy /roles authorization with custom user's claims.

**Describe alternatives you've considered**
Currently I'm using a customized version of PolicyServer.Local.

**Comment by amchew** [comment]
Hi @fileman! Could you elaborate more on what you'd like to see on policy/roles authorization with custom user's claims. For example, you can share on your use case and how user roles are authorized in your app and how that controls access to data. 

Thanks!

**Comment by amchew** [comment]
Hey @fileman, would mind sharing about your use case please? Would be very interested to learn more!

**Comment by fileman** [comment]
@amchew I've replied to the first request and now forwarded the reply to your email.

**Comment by amchew** [comment]
Hey @fileman, yes I saw that! Following up on it with the team.

**Comment by Zakeelm** [comment]
@fileman  Thank you so much for your feedback. Unfortunately, this work will not be completed. We announced this week in a [blog post]( https://devblogs.microsoft.com/appcenter/app-center-mbaas-retirement/) that we’re retiring the Auth and Data services later this year and recommending customers transition to Azure Active Directory B2C and Azure Cosmos DB instead. Let me know if you have any additional questions about this transition and i'll make sure to get them answered! Closing this issue.
